File: LeetCode-Programs/Monotonic Array.py
class Solution:
    def isMonotonic(self, nums: List[int]) -> bool:
        count_dec, count_inc=1,1
        for i in range(len(nums)-1):
            if nums[i]>=nums[i+1]:
                count_dec+=1
            if nums[i]<=nums[i+1]:
                count_inc+=1
        if count_dec==len(nums) or count_inc==len(nums):
            return True
        return False

File: LeetCode-Programs/Monotonic Array.cpp
class Solution {
public:
    bool isMonotonic(vector<int>& nums) {
        int count_inc=1, count_dec=1;
        for(int i=0;i<nums.size()-1;i++){
            if(nums[i]>=nums[i+1])
                count_dec++;
            if(nums[i]<=nums[i+1])
                count_inc++;
        }
        if(count_dec==nums.size() || count_inc== nums.size())
            return true;
        return false;
        
    }
};
